django/chat/responses.py

```diff
@@ -464,8 +472,10 @@ async def add_files_to_library():
         )()
         if error_documents:
             error_string = _("Error processing the following document(s):")
-            doc_names_for_error = [doc.filename for doc in error_documents]
-            error_docs_joined = "\n\n - " + "\n\n - ".join(doc_names_for_error)
+            doc_errors = [
+                f"{doc.filename} _{doc.status_details}_" for doc in error_documents
```

Comment:
This shows the Error IDs for documents that didn't process correctly when files are uploaded in the chat to Q&A mode.